src/main/java/com/synopsys/integration/jenkins/scan/input/NetworkAirGap.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * synopsys-security-scan-plugin
+ *
+ * Copyright (c) 2023 Synopsys, Inc.
+ *
+ * Use subject to the terms and conditions of the Synopsys End User Software License and Maintenance Agreement. All rights reserved worldwide.
+ */
+package com.synopsys.integration.jenkins.scan.input;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class NetworkAirGap {
```

Comment:
this class should be inside `com.synopsys.integration.jenkins.scan.input.airGap` package

Reply:
Airgap is only one class and it need to be available in BridgeInput which is available in "input" package. So kept it here